### PR TITLE
526 remove vcl mg

### PR DESCRIPTION
**Pull request by markgreenburg**
- Turns homelessness into a yes / no question (not required)
- Removes other questions that we can't support for MVP due to API limitations.

screen shot 2018-05-30 at 6 17 19 pm

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 1]
```diff
@@ -230,35 +229,24 @@ const formConfig = {
...
-            'view:homeless': {
-              'ui:title': 'Homeless or at risk of becoming homeless?'
-            },
-            'view:extremeFinancialHardship': {
-              'ui:title': 'Suffering from extreme financial hardship?'
+            'ui:title': 'Homelessness',
+            'view:homelessness': {
+              'ui:title': 'Are you homeless or in danger of becoming homeless?',
+              'ui:widget': 'yesNo',
+              'ui:options': {
```
Comment (cehsu) [inline src/applications/disability-benefits/526EZ/config/form.js]: Interesting, are these not default on the `yesNo` widget?

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/applications/disability-benefits/526EZ/config/form.js (hunk 1 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: Good question, I didn't look at the widget internals to find out. Probably worth double-checking, though.

Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/config/form.js]: Yep, those are definitely the defaults 😛 
Removed the whole deal.

Comment (cehsu) [reply src/applications/disability-benefits/526EZ/config/form.js]: Thanks for checking 👍

**Review by peggygannon — CHANGES_REQUESTED** [review]
@markgreenburg Hi, Mark, can we please update the homeless question to the following to match how we treat this question on the static page:
Are you homeless or at risk of becoming homeless? 
Yes
No

And I have a few followup questions:
1. what happens if the user chooses yes? does another screen pop up? 
2. Is this question on its own screen or with other info? If it's on its own screen, I'm wondering if we should have a line of text before the question. 
3.  Are the financial hardship and blind/low vision questions moving to another screen? or are they being taken out of the MVP altogether?

**Comment by markgreenburg** [comment]
@peggygannon updated content and screenshot per your recommendation.

> what happens if the user chooses yes? does another screen pop up?

@goldenmeanie 's proposed UI was to have the follow-up content info expand under these radio buttons if the veteran chooses 'yes.' This work has been offloaded to another issue because, even though simple to implement, it touches a lot of different parts of form code.

> Is this question on its own screen or with other info? If it's on its own screen, I'm wondering if we should have a line of text before the question.

Alex's proposal was to have homelessness on its own form page - with the additional name / contact info questions also on the same page, but only showing if the veteran selects 'yes.'

> Are the financial hardship and blind/low vision questions moving to another screen? or are they being taken out of the MVP altogether?

If I recall correctly, they're being taken out altogether, because there's no way to submit this info, based on the API that EVSS is providing. The only data we're able to send them is point of contact name & phone number for homeless veterans.

**Review by peggygannon — APPROVED** [review]

**Comment by markgreenburg** [comment]
Sure thing, @peggygannon ! Once we have the other PR with the follow-up contact info fields ready to go, we'll be sure to share screenshots.

**Pull request merged**